src/cli.ts

```diff
@@ -184,6 +184,7 @@ program
     '--space <space>',
     'the target Kibana spaces for the pushed monitors — spaces help you organise pushed monitors.'
   )
+  .option('-y, --yes', 'skip all questionares and do forceful push')
```

Comment:
```suggestion
  .option('-y, --yes', 'respond with the default option to all prompts, useful for CI and other automated environments')
```

I think this definition might be more future-proof, thoughts?

Reply:
Hmm, we dont default to all options like `projectid, url, space` so might need a different wording. Do you have any other suggestion?

Reply:
Reworded a bit.